net-irc/limnoria/limnoria-20200124.ebuild

```diff
@@ -1,34 +1,36 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-PYTHON_COMPAT=( python3_7 )
 
+PYTHON_COMPAT=( python3_{7..9} )
+DISTUTILS_USE_SETUPTOOLS=no
 inherit distutils-r1
 
 MY_PV="${PV:0:4}-${PV:4:2}-${PV:6:2}"
 MY_PN="Limnoria"
 MY_P="${MY_PN}-${MY_PV}"
 
-if [[ ${PV} == "99999999" ]]; then
+if [[ ${PV} == *9999 ]]; then
 	EGIT_REPO_URI="https://github.com/ProgVal/${MY_PN}.git"
 	EGIT_BRANCH="testing"
 	inherit git-r3
-	SRC_URI=""
 else
+	S="${WORKDIR}/${MY_PN}-master-${MY_PV}"
 	SRC_URI="https://github.com/ProgVal/${MY_PN}/archive/master-${MY_PV}.tar.gz -> ${P}.tar.gz"
 	KEYWORDS="~amd64 ~x86"
-	S="${WORKDIR}/${MY_PN}-master-${MY_PV}"
 fi
 
 DESCRIPTION="Python based extensible IRC infobot and channel bot"
 HOMEPAGE="https://supybot.aperio.fr/"
```

Comment:
Not their HOMEPAGE anymore (and Limnoria is a fork of supybot)

Reply:
Thanks, I wasn't checking

Reply:
I'm an IRC obsessive which is why I know :)